packages/vc.js/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@transmute/vc.js",
   "author": "Orie Steele",
   "module": "dist/vc.js.esm.js",
-  "version": "0.2.1-unstable.9",
+  "version": "0.2.1-unstable.10",
```

Comment:
you shouldn't bump the version manually, it's going to be done automatically here: https://github.com/transmute-industries/vc.js/blob/master/.github/workflows/cd.yml#L39

Reply:
Ok, should be fixed now